nucypher/blockchain/eth/interfaces.py

```diff
@@ -72,9 +88,9 @@ class BlockchainInterface:
                       }
 
     process = NO_PROVIDER_PROCESS.bool_value(False)
-    Web3 = Web3
+    Web3 = Web3  # TODO: This is name-shadowing the actual Web3. Is this intentional?
```

Comment:
Does it actually do anything at all?

Reply:
At first blush - No I dont think so.  @vepkenez ?

Reply:
Ah, my bad, it creates a class attribute.